dyninstAPI/h/BPatch_snippet.h

```diff
@@ -238,8 +238,10 @@ class BPATCH_DLL_EXPORT BPatch_constExpr : public BPatch_snippet {
     //  Creates a representation of a (signed long) value
     BPatch_constExpr(signed long value);
     
-    //  Creates a representation of an (unsigned long) value
```

Comment:
The basic problem is that sizeof(void*) == unsigned long everywhere except for Win64, where sizeof(void*) == unsigned long long.
I'm open to better ways to handle this, uintptr_t just seemed like the most cross-platform way that immediately came to mind.